pydoctor/test/test_sphinx.py

```diff
@@ -636,6 +636,14 @@ def test_prepareCache(
     L{cachecontrol.CacheControlAdapter} for C{http} and C{https} URLs.
     """
 
+    # Windows doesn't like paths ending in a space or dot.
+    assume(cacheDirectoryName[-1] not in '. ')
+
+    # These DOS device names still have special meaning in modern Windows.
+    assume(cacheDirectoryName.upper() not in {'CON', 'PRN', 'AUX', 'NUL'})
+    assume(not cacheDirectoryName.upper().startswith('COM'))
+    assume(not cacheDirectoryName.upper().startswith('LPT'))
```

Comment:
I guess that is ok...but I think this is not the right way to fix it.

Instead of st.text() I think that we should have an st.path or some helper which will always return a valid path.

Reply:
I looked for a file name generating strategy in Hypothesis' docs, but couldn't find any. One could be written of course, but I think that's overkill for the one test case in pydoctor that uses this strategy.

The Hypothesis docs do suggest that as long as `assume()` rejects only a small portion of the generated values, it's a valid approach.

Reply:
Yep. Thanks.